### PR TITLE
feat: Relax naming and complexity rules

### PR DESCRIPTION
**Pull request by mingnz**

### CODE REVIEW

File: lib/common.js [hunk 1]
```diff
@@ -75,16 +74,7 @@ module.exports = {
...
-      "error",
-      {
-        arrays: "always-multiline",
-        objects: "always-multiline",
-        imports: "never",
-        exports: "never",
-        functions: "never",
-      },
-    ],
+    "comma-dangle": ["error", "always-multiline"],
```
Comment (danieljimeneznz) [inline lib/common.js:77]: Why this change? Curious on the `functions` one?

Comment (mingnz) [reply lib/common.js:77]: If we are going to trailing comma, might as well trailing comma everything.

Okay I'll come clean, it's how `prettier` behaves by default 😅 

I'll let you decide whether you want to incorporate it

Comment (danieljimeneznz) [reply lib/common.js:77]: I just don't want functions to do:

```
/*eslint comma-dangle: ["error", {"functions": "always"}]*/

function foo(a, b,) {
}

foo(a, b,);
new foo(a, b,);
```

**Review by danieljimeneznz — APPROVED** [review]
